resources/python/ci/dead_code_analysis.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -eu
+
+vulture --min-confidence 95 "griffin"
```

Comment:
Error: "griffin" package doesn't exist. Not in the Cura repository anyway.

Was this meant to be specific to certain repositories? Perhaps it's better to mark it as such or order them in folders as such.